timemachine/md/exchange/exchange_mover.py

```diff
@@ -13,6 +13,17 @@
 from timemachine.potentials import nonbonded
 
 
+def get_water_idxs(mol_groups: List[NDArray], ligand_idxs: Optional[NDArray] = None) -> List[NDArray]:
+    """Given a list of lists that make up the individual molecules in a system, return the subset that is only the waters.
+    Additional logic to handle the case where ligand_idxs is also of size 3.
```

Comment:
Additional logic is present or is needed (to handle cases where ligand_idxs == 3?)

Reply:
Is present, will clarify.

Reply:
Clarified in https://github.com/proteneer/timemachine/pull/1286/commits/8fd9a326950e594cf1fc18aa177e1bad5729e724